src/main/java/app/chatbot/Duke.java

```diff
@@ -0,0 +1,92 @@
+package app.chatbot;
+
+import java.io.IOException;
+import java.nio.file.Path;
+import java.nio.file.Paths;
+
+import app.command.Command;
+import app.command.LoadCommand;
+import app.command.SaveCommand;
+import app.task.TaskList;
+
+public class Duke {
+    private static final Path STORAGE_LOCATION = Paths.get(".", "data", "storage.txt");
+
+    private Storage storage;
+    private TaskList taskList;
+    private Ui ui;
+
+
+    public Duke() {
+        this.ui = new Ui();
+        try {
+            this.storage = new Storage(this.STORAGE_LOCATION);
```

Comment:
`STORAGE_LOCATION` is a static variable and should not be accessed using the Object variable identifier `this`. It should be accessed using the class name `Duke` instead or just simply accessed without any identifier, `STORAGE_LOCATION`.
```suggestion
            this.storage = new Storage(Duke.STORAGE_LOCATION);
```